client/app/utils/jsonSchema/operations.ts

```diff
@@ -415,12 +416,27 @@ const operationPage2: RJSFSchema = {
   ],
 };
 
+const operationPage3: RJSFSchema = {
+  type: "object",
+  title: "Statutory Declaration and Disclaimer",
+  // Uncomment this once documents are implemented
+  /*   required: ["statutory_declaration"], */
```

Comment:
Wasn't sure if I should leave this in for development purposes. Developers would still be able to get around this if we left it required by added a file even if nothing would happen on the backend.